Content/ReleaseBriefings/4_6/Section_GeneralEnhancements.tex

```diff
@@ -0,0 +1,154 @@
+  With the Cuda and HIP backends, \texttt{Kokkos::Experimental::inclusive\_scan} now calls the vendor versions in Thrust
+  \begin{itemize}
+     \item The vendor versions are faster than the \texttt{Kokkos::parallel\_scan}-based default implementation
+     \item Thrust requires \texttt{Kokkos\_ENABLE\_ROCTHRUST} to be \texttt{ON} (which is the default)
+  \end{itemize}
+\end{frame}
+
+\begin{frame}[fragile]{Reduce tooling interface overhead}
+  Reduced the overhead of Kokkos tools related checks
+  \begin{itemize}
+      \item Store the information whether Kokkos tools are enabled after each modification to the tools' callbacks
+      \item Previously, this value was recomputed for every event (\texttt{parallel\_for}, \texttt{fence}, etc.)
+      \item Most noticeable in small serial kernels (around 100 elements)
+  \end{itemize}
+\end{frame}
+
+%==========================================================================
+\begin{frame}[fragile]{SIMD reductions and compound assignments}
+  Added reductions and remaining compound assignments to Kokkos SIMD
+
+  \begin{itemize}
+    \item \texttt{basic\_simd\& operator/=(basic\_simd\&, U\&\&)}
+    \item \texttt{basic\_simd\& operator>>=(basic\_simd\&, U\&\&)}
+    \item \texttt{basic\_simd\& operator<<=(basic\_simd\&, U\&\&)}
+  \end{itemize}
+
+  \vspace{5pt}
+  
+  \begin{itemize}
+    \item \texttt{T reduce\_min(const basic\_simd\& x)}
+    \item \texttt{T reduce\_max(const basic\_simd\& x)}
+    \item \texttt{T reduce(const basic\_simd\& x, const mask\_type\& mask, T identity\_element, BinaryOperation binary\_op)}
+      \begin{itemize}
+        \item Supported binary operations are: std::plus, std::multiplies, std::bit\_and, std::bit\_or and std::bit\_xor
+        \item std::plus is used if binary op is not specified
+      \end{itemize}
+  \end{itemize}
+
+\end{frame}
+%==========================================================================
+\begin{frame}[fragile]{Avoid storing a view in RandomAccessIterator}
+
+  In Kokkos 4.5, \texttt{Kokkos::sort} was changed to use raw pointers on host
+  \begin{itemize}
+    \item Resolves performance issue of \texttt{Kokkos::sort} on host when using \texttt{RandomAccessIterator}
+    \item Caused by reference counting and view access in \texttt{RandomAccessIterator}
+  \end{itemize}
+
+  \vspace{10pt}
+  
+  Following revisions are made in Kokkos 4.6
+  \begin{itemize} 
+    \item Reverted previous changes in \texttt{Kokkos::sort}
+    \item Replaced views with raw pointers in \texttt{RandomAccessIterator} instead
+    \begin{itemize}
+      \item Prevents \texttt{RandomAccessIterator} from participating in view reference counting
+    \end{itemize}
+  \end{itemize}
+
+\end{frame}
+%==========================================================================
+\begin{frame}[fragile]{Print support for system allocated memory}
+  \texttt{print\_configuration} outputs if system allocated memory is accessible on GPU
```

Comment:
Fine but to be clear we do not make any promise on the format

Reply:
done in https://github.com/kokkos/kokkos-tutorials/pull/125/commits/bc29a03d2e6c3ac4529bf8243849587170e0f9b9